dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -165,39 +165,12 @@ public static String replaceExtension(String originalName, String newExtension)
     public static String getFacetFileType(DataFile dataFile) {
         String fileType = dataFile.getContentType();
 
-        if (!StringUtil.isEmpty(fileType)) {
-            if (fileType.contains(";")) {
-                fileType = fileType.substring(0, fileType.indexOf(";"));
-            }
-
-            try {
-                return BundleUtil.getStringFromPropertyFile(fileType, "MimeTypeFacets");
-            } catch (MissingResourceException e) {
-                // if there's no defined "facet-friendly" form of this mime type
-                // we'll truncate the available type by "/", e.g., all the 
-                // unknown image/* types will become "image"; many other, quite
-                // different types will all become "application" this way - 
-                // but it is probably still better than to tag them all as 
-                // "uknown". 
-                // -- L.A. 4.0 alpha 1
-                //
-                // UPDATE, MH 4.9.2
-                // Since production is displaying both "tabulardata" and "Tabular Data"
-                // we are going to try to add capitalization here to this function
-                // in order to capitalize all the unknown types that are not called
-                // out in MimeTypeFacets.properties
-                String typeClass = fileType.split("/")[0];
-                return Character.toUpperCase(typeClass.charAt(0)) + typeClass.substring(1);
```

Comment:
We don't need this anymore? It looks like they didn't want all friendly-facet missing values to be "Unknown".

Reply:
It didn't work this way and nobody noticed but I guess we can make this work 'as intended' :) 
For the "tabular data on production case" I would skip this. If we ever have similar issue I would fix it in some other way.

Reply:
Well if we didn't need this before we can leave it then, if there is going to be need for new type of behaviour in tag naming, we can implement it then.